the-blue-alliance-ios/Core Data/Match/Match.swift

```diff
@@ -169,6 +169,15 @@ extension Match {
     var blueAllianceTeamNumbers: [String] {
         return blueAllianceTeamKeys.map({ Team.trimFRCPrefix($0) })
     }
+    
+    /**
+     Returns the team keys from both alliances
+    */
+    var teamKeys: [TeamKey]? {
+        guard let blueKeys = blueAlliance?.teams?.array as? [TeamKey] else { return nil }
```

Comment:
I think we can do better here - use the `alliances` property, which looks like a set of `MatchAlliance` objects. And then we should be able to flatten the two sets in to one.

Also, these methods usually return empty objects instead of nulls - see `dqTeamKeys` for example.

Reply:
Also - can we make sure this gets tests? All the Core Data stuff is pretty well tested - I wanna make sure we keep test coverage high on the models.

Reply:
For the test, how should I go about checking the array of Team Keys? Currently, I just checked to see if the length of the teamKeys is equal to 6 or should I go through each index and see if the whole object of Team Key checks out?

Reply:
I would check the keys for sure - here's an example of where we do that in other tests https://github.com/the-blue-alliance/the-blue-alliance-ios/blob/2c4d804e196292309774e9e4cbf3d27ccaa45391/tba-unit-tests/Core%20Data/Match/MatchTests.swift#L477